src/sidebar/store/modules/links.js

```diff
@@ -22,6 +22,13 @@ function updateLinksAction(newLinks) {
   return { type: 'UPDATE_LINKS', newLinks: newLinks };
 }
 
+/**
+ * @typedef LinksStore
+ *
+ * // actions
+ * @prop {typeof updateLinks} updateLinksAction
+ */
+
```

Comment:
Line 35: I'm a bit confused what `update: { UPDATE_LINKS: updateLinks }` is supposed to be
I'll take a fresh look at that later on, but something does not seem right to me here...